docs/source/protocol_v3.rst

```diff
@@ -1,51 +1,437 @@
+state of the simulation. Instead, subsequent pause packets sent after the first one will have no effect.
+
+.. code-block:: none
+
+   Header:
+      7 (int32) Pause
+      <val> (no type) Unused slot, any value acceptable
+
+
+Transmission rate
+^^^^^^^^^^^^^^^^^
+
+Sent from the receiver to the simulation engine any time after the `Session info`_
+has been sent to change the number of steps between each IMD frame. For example,
+if the transmission rate is 1, every integration step in the simulation will 
+be an IMD frame.
+
+.. code-block:: none
+
+   Header:
+      8 (int32) Transmission rate
+      <val> (int32) New transmission rate. Any value less than 1 will reset 
+                    the transmission rate to its default value (configured
+                    by the simulation engine)
+
+IO Error
+^^^^^^^^
+
+Never sent from one party to another during an IMD session. Can be used internally 
+by the simulation engine or receiver to indicate an error has occurred.
+
+.. code-block:: none
+
+   Header:
+      9 (int32) IO Error
+      <val> (no type) Unused slot, any value acceptable
+
+
+Session info
+^^^^^^^^^^^^
+
+Sent by the simulation engine to the receiver immediately after
+the `Handshake`_ is sent to indicate to the receiver which data it 
+should expect for each IMD frame during the session along with
+whether coordinates will be wrapped into the simulation box if present.
+
+.. code-block:: none
+
+   Header:
+      10 (int32) Session info
+      7 (int32) Number of 1-byte configuration options in the body packet
     
-    v. Force data (IMD_FORCES) in the same manner as positions
+   Body:
+      <val> (int8) Nonzero if time packets sent in each IMD frame
+      <val> (int8) Nonzero if IMD energy block packets sent in each IMD frame
+      <val> (int8) Nonzero if box packets sent in each IMD frame
+      <val> (int8) Nonzero if coordinate packets sent in each IMD frame 
+      <val> (int8) Nonzero if coordinates wrapped into the simulation box. 
+                   Meaningless if coordinates not sent in the session
+      <val> (int8) Nonzero if velocity packets sent in each IMD frame 
+      <val> (int8) Nonzero if force packets sent in each IMD frame 
+
+Resume
+^^^^^^
+
+Sent from the receiver to the simulation engine any time after the `Session info`_
+has been sent to request that the simulation resumes execution
+if it is in a paused state. Like `Pause`_, resume is idempotent.
+
+.. code-block:: none
+
+   Header:
+      11 (int32) Resume
+      <val> (no type) Unused slot, any value acceptable
+
+Time
+^^^^
+
+Sent from the simulation engine to the receiver each IMD frame if 
+time packets were previously specified for this session in `Session info`_.
+
+.. code-block:: none
+
+   Header:
+      12 (int32) Time
+      1 (int32) Number of time packets being sent
+
+   Body:
+      <val> (unsigned float64) dt for the simulation
+      <val> (unsigned float64) Current time of the simulation
```

Comment:
Should we relax and use signed float?

Reply:
Let's discuss tomorrow. I have no strong feelings on this

Reply:
resolved to use signed float64